Tests/Spec/tentacle_scenario_01_install_spec.rb

```diff
@@ -11,12 +11,12 @@
 describe file('C:/Program Files/Octopus Deploy/Tentacle/Tentacle.exe') do
   it { should be_file }
 end
-
+=begin
```

Comment:
Whats the reasoning for these?

Reply:
The registry tests were failing both locally and on TeamCity.  Jason said it was because the work to make the Tentacle Linux compatible removed them and I should just comment them out.

Reply:
If we remove something, we should delete the code - commenting it out leaves cruft around and its hard for someone in the future to know why they are commented out.

In this case, this needs to be backwards compatible, so for older versions of tentacle, we should still be checking for the registry entries, but not for newer versions. We should check for the files on disk instead.

Reply:
Sorry it's been so long, I'm having to learn a bit if Ruby to implement some of what you were looking for.  For the newer versions you mentioned that we should look for files on disk, where on disk should I be looking?

Reply:
Here?  C:\ProgramData\Octopus\Tentacle\Instances

Reply:
I've added a version check and an if statement to handle this.  If the tentacle is an older version, it will check for the existence of the registry entries, if newer, it will check the file locations.

The ruby code used for the version check was not written by me.  My limited ruby knowledge doesn't fully understand all that it is doing so if there is something that is not right with it, I may need assistance.